scripts/lib/install.sh

```diff
@@ -0,0 +1,63 @@
+install_aws_lb_controller() {
```

Comment:
This file involves helm chart installation/deletion. Maybe rename it to helm.sh. Then the functions like:
helm::install...